### DISCUSSION TITLE
Syslog logging

### DISCUSSION THREAD

**Discussion by OH2LAK** [lora-aprs/LoRa_APRS_iGate#146] [Ideas]
We are to use the TTGO ethernet boards to build our LoRa-APRS infrastructure for digi's and igates and thus the serial logging feature is not enough. Could a syslog logging feature be implemented to be able to do remote logging?

Looks like there is a suitable Arduino syslog library available ( https://github.com/arcao/Syslog ) which could do the job. I'm not a programmer but I'm tempted to fork the project and try on my own with learning by error.

@peterus Could you consider adding this to the igate code if I fail miserably?

Thanks for a great project and code!
Erik OH2LAK

**Comment by peterus** [comment]
Hi Erik!
What a great idea! I really like it! First I was thinking logging to a SD card would be enough, but of course if you have multiple sites it is really hard to get all log files.
My plan would be to integrate the syslog logging into my [logging library](https://github.com/peterus/esp_logger). Than the changes  to the iGate are not so much.
Further I will create a public syslog-ng server with [webinterface](https://lggr.io/) that people which don't want to host the server by themself have a possibility. If you know better services (log server or webinterface) for a public server please let me know!
I do not have currently so much time, so give me some days. If you want, you can start of course ;)
